xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -759,8 +759,11 @@ void CGUIDialogPVRTimerSettings::InitializeTypesList()
     if (type->ForbidsEpgTagOnCreate() && m_timerInfoTag->GetEpgInfoTag())
       continue;
 
-    // Drop TimerTypes that aren't rules if end time is in the past
-    if (!type->IsTimerRule() && m_timerInfoTag->EndAsLocalTime() < CDateTime::GetCurrentDateTime())
+    EPG::CEpgInfoTagPtr epgTag = m_timerInfoTag->GetEpgInfoTag(false);
```

Comment:
const EPG::CEpgInfoTagPtr epgTag(m_timerInfoTag->GetEpgInfoTag(false)); if you don't mind